python/foglamp/plugins/south/http_south/http_south.py

```diff
@@ -148,16 +179,11 @@ class HttpSouthIngest(object):
                         "timestamp": "2017-01-02T01:02:03.23232Z-05:00",
                         "asset": "pump1",
                         "key": "80a43623-ebe5-40d6-8d80-3f892da9b3b4",
-                        "readings": {
-                            "velocity": "500",
-                            "temperature": {
-                                "value": "32",
-                                "unit": "kelvin"
-                            }
+                        "readings": {"humidity": 0.0, "temperature": -40.0}
```

Comment:
was there a problem in old docstring sample?

Reply:
Just to make the example shorter and clearer.